docs/installation.rst

```diff
@@ -9,7 +9,7 @@ Minimum Requirements
 - ShotGrid server v5.4.14 or higher for API v3.0.16+.
```

Comment:
Did we say that we need to remove these 3 lines? (or will it be part of another initiative?)